uv.lock

```diff
@@ -581,25 +582,25 @@ wheels = [
 
 [[package]]
 name = "fake-useragent"
-version = "1.5.1"
+version = "2.0.0"
```

Comment:
@sam-mellor @iopapamanoglou does this need more scrutiny?

Reply:
opened #648